extensions/amp-analytics/0.1/vendors/deepbi.json

```diff
@@ -5,7 +5,7 @@
     "endpoint": "api.deep.bi"
   },
   "requests": {
-    "base": "/v1/pixel/${datasetKey}/p.gif?accessKey=${accessKey}&gdpr=${gdpr}&page_href=${sourceUrl}&referrer_href=${documentReferrer}&page_title=${title}&amp=true",
+    "base": "/v1/pixel/${datasetKey}/p.gif?accessKey=${accessKey}&gdpr=${gdpr}&page_href=${sourceUrl}&referrer_href=${documentReferrer}&page_title=${title}&deepbiampid={clientId(deepbi_amp)}&amp=true",
```

Comment:
`{clientId(deepbi_amp)}` => `${clientId(deepbi_amp)}`

So that the macro triggers.

Reply:
Yeah, sure. I will correct that.